src/backend/api/Fusion.Resources.Api/Controllers/Requests/AnalyticsController.cs

```diff
@@ -70,6 +70,33 @@ public async Task<ActionResult<ApiCollection<ApiPersonAbsenceForAnalytics>>> Get
             return collection;
         }
 
+        [MapToApiVersion("2.0")]
+        [HttpGet("/analytics/absence/internal")]
+        public async Task<ActionResult<ApiCollection<ApiPersonAbsenceForAnalyticsV2>>> GetPersonsAbsenceV2([FromQuery] ODataQueryParams query)
+        {
+            #region Authorization
+
+            var authResult = await Request.RequireAuthorizationAsync(r =>
+            {
+                r.AlwaysAccessWhen().FullControl().FullControlInternal();
+                r.AnyOf(or =>
+                {
+                    or.GlobalRoleAccess("Fusion.Analytics.Requests");
```

Comment:
Should be "Fusion.Analytics.Absence" (for both V1&V2 endpoint)
Needs to be defined in Azure AD.